Added axes-based controls and movement

Calculus-Master opened the pull request.

Calculus-Master committed:
Added axes-based controls and movement

Calculus-Master committed:
Added axes-based controls and movement

Sajo8 reviewed (changes requested):
revert changes made to compile_commands.json

Calculus-Master committed:
Added functioning axes controls [Read desc]

- Added functioning axes controls, removed all button controls
- Added button controls to modify the velocity in real time, renamed the masterStop button to minVel
- Removed turnLeft and turnRight; the axes can do that if you use both at the same time
- Added a catch to the setTotalVelocity function so if the parameter is negative the motors will just stop instead
- It all works!

Sajo8 committed:
revert changes made to compile_commands.json
